popup.js

```diff
@@ -201,16 +201,18 @@ function runThePopup() {
     }
   });
   
-  
   chrome.tabs.query({
       active: true,
       currentWindow: true
-  }, loadTabData);
+  }, onTabs);
   
+  function onTabs(tabs) {
+	 chrome.storage.sync.get(['videoID'], function(result) {
+         	loadTabData(tabs, result.videoid);
+	 }); 
```

Comment:
What is the point of this, why doesn't it send a message to the tab asking for the videoID?

Reply:
There might be problems when switching between multiple tabs

- Make Storage API contain the tabID
- Send a message for the active tab (Probably best option)

Reply:
Went for "Send a message to active tab"